src/components/__tests__/actions/forgotPasswordActions.test.js

```diff
@@ -0,0 +1,160 @@
+import configureStore from 'redux-mock-store'; // ES6 modules
+import thunk from 'redux-thunk';
+import nock from 'nock';
+import 'isomorphic-fetch';
+import {
+  submitForgotPasswordFailure,
+  submitForgotPasswordSuccess,
+  handleForgotPasswordForm,
+  submitResetPassword,
+  submitForgotPassword,
+} from '../../../redux/actions/forgotPasswordActions';
+import {
+  FORGOT_PASSWORD_FORM,
+  FORGOT_PASSWORD_SUCCESS,
+  FORGOT_PASSWORD_FAILURE,
+  SUBMIT_FORGOT_PASSWORD_FORM,
+} from '../../../redux/actions-types/forgotPasswordTypes';
+
+const { API_URL = 'http://localhost:3000/api/v1' } = process.env;
+
+const mockStore = configureStore([thunk]);
+let store;
+jest.setTimeout(30000);
+
+describe('forgotPassword', () => {
+  describe('synchronous actions', () => {
+    beforeEach(() => {
+      store = mockStore({});
+    });
+    test('should dispatch FORGOT_PASSWORD_SUCCESS action', () => {
+      const payload = { field: 'newPassword', value: '123456' };
+      const expectedPayload = {
+        type: FORGOT_PASSWORD_FORM,
+        payload,
+      };
+
+      store.dispatch(handleForgotPasswordForm(payload));
+
+      const actions = store.getActions();
+      expect(actions).toEqual([expectedPayload]);
+    });
+    test('should dispatch FORGOT_PASSWORD_SUCCESS action', () => {
```

Comment:
space this test

Reply:
@kimotho-njoki I have added an empty line before test (line 42).